client/src/components/backend/Form/HasMany.js

```diff
@@ -10,7 +10,8 @@ export default class FormMakers extends PureComponent {
 
   static propTypes = {
     label: PropTypes.string.isRequired,
-    onNew: PropTypes.func.isRequired,
+    includeCreate: PropTypes.bool,
```

Comment:
Do we really need includeCreate? Does it make sense to act as thought includeCreate == false if no onNew prop is passed? What would happen if includeCreate == true and there's an onNew callback? Seems like we're setting two switches where we only need one. 

We should also keep our naming consistent. Is "new" the same thing as "create"? If we do end up needing two switches here, I think we should call them "onNew" and "allowNew"